.github/workflows/test-unit.yaml

```
@@ -0,0 +1,32 @@
+name: test-unit
+
+on:
+  - pull_request
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+      - uses: pre-commit/action@v3.0.1
+
+  coverage:
+    runs-on: ubuntu-latest
```

Can you call this job `test`, like we did here: https://github.com/kubeflow/trainer/blob/master/.github/workflows/test-python.yaml#L14
```suggestion
  test:
    name: Test
    runs-on: ubuntu-latest
```